src/services/jwt.service.js

```
@@ -0,0 +1,38 @@
+import jwt from 'jsonwebtoken';
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
```

There is no guard that `process.env.JWT_KEY` / `process.env.JWT_REFRESH_KEY` exist before using them. If these env variables are missing, signing will throw at runtime. Consider checking these at startup or throwing a descriptive error here to fail fast and make debugging easier.